native/src/main/java/io/ballerina/stdlib/oracledb/parameterprocessor/OracleDBResultParameterProcessor.java

```diff
@@ -132,4 +137,46 @@ public BObject getBalStreamResultIterator() {
         }
         return struct;
     }
+
+    @Override
+    public Object processXmlResult(ResultSet resultSet, int columnIndex, int sqlType, Type ballerinaType)
+            throws ApplicationError, SQLException {
+        try {
+            SQLXML sqlxml = resultSet.getSQLXML(columnIndex);
+            return this.convertXml(sqlxml, sqlType, ballerinaType);
+        } catch (NoClassDefFoundError e) {
+            throw new ApplicationError("Error occurred while retrieving an xml data. Check whether both " +
+                    "`xdb.jar` and `xmlparserv2.jar` are present in the dependency jar list");
```

Comment:
```suggestion
                    "`xdb.jar` and `xmlparserv2.jar` are added correctly in Ballerina.toml");
```

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-oracledb/pull/10/commits/e325a68ee55c9bd5a7b17dbad611333730c549df